### PR TITLE
Education C1

### PR DESCRIPTION
**Pull request by Nickelody**
[    {"EN":{        "word": "advancement",        "topics": ["science"],	"level": "advanced(C1)"        }},    {"RU": {        "word": "продвижение",        "topics": ["наука"]        }}    ], 	[    {"EN":{        "word": "analyse",        "topics": ["science"],	"level": "intermediate(B1)"        }},    {"RU": {        "word": "анализировть",        "topics": ["наука"]        }}    ],     	[    {"EN":{        "word": "case study",        "topics": ["science"],        "level": "upper-intermediate(B2)"        }},    {"RU": {        "word": "тематическое исследование",        "topics": ["наука"]        }}    ]

### CODE REVIEW

File: app/src/main/res/raw/data.json [hunk 1]
```diff
@@ -8238,4 +8238,37 @@
...
+	"level": "intermediate(B1)"
+        }},
+    {"RU": {
+        "word": "анализировть",
+        "topics": ["наука"]
+        }}
+    ],     
+	[
+    {"EN":{
+        "word": "case study",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:8265]: please remove because it is 2 words

File: app/src/main/res/raw/data.json [hunk 2]
```diff
@@ -8238,4 +8238,37 @@
         "topics": ["образование"]
         }}
     ]
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:8240]: please add comma here

File: app/src/main/res/raw/data.json [hunk 3]
```diff
@@ -8237,5 +8237,27 @@
...
-    ]
+    ],
+	[
+    {"EN":{
+        "word": "advancement",
+        "topics": ["science"],
+	"level": "advanced(C1)"
+        }},
+    {"RU": {
+        "word": "продвижение",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:8248]: исправь пожалуйста на развитие

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

File: app/src/main/res/raw/data.json [hunk 4]
```diff
@@ -8231,11 +8216,265 @@
...
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "department",
+            "topics": ["education"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "кафедра",
```
[the remark this answers is not in the document]
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:8322]: discussed offline - кафедра ok

**Review by omen273 — APPROVED** [review]

**Pull request merged**